Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -102,41 +90,25 @@ public final class ProjectPamphletViewController: UIViewController {
     }
   }
 
-  private func configureViews() {
-    // Configure subviews
-    self.backThisProjectContainerView.addSubview(self.backThisProjectButton)
-
-    self.view.addSubview(self.backThisProjectContainerView)
-
-    self.backThisProjectButton.addTarget(self, action: #selector(backThisProjectTapped), for: .touchUpInside)
-
-    // Configure constraints
-    let backThisProjectContainerViewConstraints = [
-      self.backThisProjectContainerView.leftAnchor.constraint(equalTo: self.view.leftAnchor),
-      self.backThisProjectContainerView.rightAnchor.constraint(equalTo: self.view.rightAnchor),
-      self.backThisProjectContainerView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor)
-    ]
-
-    let containerMargins = self.backThisProjectContainerView.layoutMarginsGuide
-    let minHeight = Styles.minTouchSize.height
-
-    let backThisProjectButtonConstraints = [
-      self.backThisProjectButton.leftAnchor.constraint(equalTo: containerMargins.leftAnchor),
-      self.backThisProjectButton.rightAnchor.constraint(equalTo: containerMargins.rightAnchor),
-      self.backThisProjectButton.bottomAnchor.constraint(equalTo: containerMargins.bottomAnchor),
-      self.backThisProjectButton.topAnchor.constraint(equalTo: containerMargins.topAnchor),
-      self.backThisProjectButton.heightAnchor.constraint(greaterThanOrEqualToConstant: minHeight)
-    ]
-
-    NSLayoutConstraint.activate(backThisProjectContainerViewConstraints + backThisProjectButtonConstraints)
-  }
-
   public override func bindStyles() {
     super.bindStyles()
 
     _ = self.backThisProjectContainerView
       |> \.layoutMargins .~ .init(all: backThisProjectContainerViewMargins)
 
+    _ = self.backThisProjectContainerView.layer
+      |> \.backgroundColor .~ UIColor.white.cgColor
+      |> \.shadowColor .~ UIColor.black.cgColor
+      |> \.shadowOpacity .~ 0.12
+      |> \.shadowOffset .~ CGSize(width: 0, height: -1.0)
+      |> \.shadowRadius .~ 1.0
+      |> \.cornerRadius .~ CheckoutStyleConstants.cardStyleRoundedCornerRadius
+
+    if #available(iOS 11.0, *) {
+      _ = self.backThisProjectContainerView.layer
+        |> \.maskedCorners .~ [.layerMaxXMinYCorner, .layerMinXMinYCorner]
+    }
```

Comment:
No `else` because we're removing iOS 10 support imminently.

Reply:
🔜

Reply:
We can potentially remove this now since the iOS 10 deprecation has been merged to `master` and should have been merged to `feature-native-checkout`

Reply:
They're removed here: https://github.com/kickstarter/ios-oss/pull/679